go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/go-gota/gota
+
+go 1.12
+
+require gonum.org/v1/gonum v0.6.2
```

Comment:
Please don't add this here, as is not needed for the PR

Reply:
yes noted